CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+- Added Update-VSTeamGitRepositoryDefaultBranch to allow for changing the default branch of a repository.
```

Comment:
```suggestion
## 7.8.0

- Added Update-VSTeamGitRepositoryDefaultBranch to allow for changing the default branch of a repository.
```

Reply:
For a new freature you can raise the minor version. Can you please do the same in the PS manifest?